### DISCUSSION TITLE
postgresml compatibility

### DISCUSSION THREAD

**Discussion by louis030195** [supabase/supabase#14046] [Feature Requests] [answered]
https://github.com/postgresml/postgresml

I assume that supabase infra does not support gpus or ml-specialised hardware as of now but could be cool even on cpu 😛 

can i install it in my instance or how do we add extension to supabase? happy to send a pr

**Comment by inian** [comment, marked as answer]
Please vote for the extension [here](https://github.com/orgs/supabase/discussions/679#discussioncomment-6042655) so that we can prioritise with the rest of the extensions
